File: Add Code Here/C++/prime.cpp
#include<stdio.h>
int main()
{
	int n,i,res=1;
	printf("Enter number: ");
	scanf("%d",&n);
	for(i=2;i<n;i++)
	{
		if(n%i==0)
		{
			res=i;
		}
	}
	if(res==1)
 	{
 		printf("prime");
	 }
	else
	{
 		printf("not a prime");
	 }
}

File: Add Code Here/GoLang/prime.go
package main

import "fmt"

func isPrime(n int) {
	for i := 2; i < n/2+1; i++ {
		if n%i == 0 {
			fmt.Println("Not a Prime Number")
			return
		}
	}
	fmt.Println("Prime Number")
	return
}

func main() {
	var n int
	_, err := fmt.Scanf("%d", &n)
	if err != nil {
		fmt.Println(err)
	}

	isPrime(n)
}
